addons/web_editor/static/src/scss/web_editor.frontend.scss

```diff
@@ -24,6 +24,17 @@
     transform: scale(-1);
 }
 
+// Needed to prevent the emoji picker from inheriting theme colors
+.o-EmojiPicker {
+    --o-EmojiPicker-bg: transparent;
```

Comment:
Why `transparent`? 🤔
Since we hardcoded other values (text color, active states, section's bg... ) I think that the main background should be hard coded as well, no? 
| this PR | expected result |
|--------|--------|
| ![image](https://github.com/odoo/odoo/assets/8558351/641b8eb2-39c2-4ab9-8d2b-5ef012b4a7af) | ![image](https://github.com/odoo/odoo/assets/8558351/e3be0fc8-e3d0-42f9-b584-81b3b2d477e4) |